SPECS/conda/conda.spec

```diff
@@ -185,7 +192,7 @@ install -m 0644 -Dt %{buildroot}%{bash_completionsdir}/ %SOURCE1
 
 %check
 %if 0%{?with_check}
-pip3 install archspec iniconfig flask pytest-xprocess zstandard conda-package-streaming flaky pytest-timeout
+pip3 install archspec iniconfig flask pytest-xprocess zstandard conda-package-streaming flaky pytest-timeout boltons
```

Comment:
You shouldn't need to pip install archspec, conda-package-streaming, menuinst, boltons archspec at this point